### PR TITLE
New real-time vehicle position fields

### PR DESCRIPTION
**Pull request by nighthawk**
See diff from YAML for details.

### CODE REVIEW

File: docs/specs/tripgo.swagger.yaml [hunk 1]
```diff
@@ -2947,6 +2947,12 @@ definitions:
         required:
           - lat
           - lng
+      speed:
+        type: number
+        description: Speed of this vehicle at the provided location, in metres per second
+      distanceTravelled:
+        type: number
+        description: Distance travelled of this vehicle since start of the journey, in metres
```
Comment (fersagui) [inline docs/specs/tripgo.swagger.yaml:2955]: Just a comment: In the GTFS-R specs it is not clear if the “distance” is from the start of the journey or between stops. It is just the odometer of the vehicle but the specs is not clear where is the zero value.

**Review by fersagui — APPROVED** [review]
LGTM. Just a simple comment about the “distanceTravelled” value

File: docs/specs/tripgo.swagger.yaml (hunk 1 above)
Comment (nighthawk) [reply docs/specs/tripgo.swagger.yaml:2955]: Ah, ok, then `distanceTravelled` is maybe not the best name and we should just name it `odometer`?

Comment (nighthawk) [reply docs/specs/tripgo.swagger.yaml:2955]: Done here: https://github.com/skedgo/skedgo-java/pull/4316

Comment (fersagui) [reply docs/specs/tripgo.swagger.yaml]: Yes. Let’s do that for now unless there is an update on the GTFS-R specs.

**Pull request merged**